doc/cache.rst

```diff
@@ -20,6 +19,9 @@ a lower version number is found the cache is recreated with the current
 version. If a higher version number is found the cache is ignored and left as
 is.
 
+The rest of this chapter describes the data structures for the local cache with
+version 1.
```

Comment:
This was already here, I mostly moved it around (and rephrased it a tiny bit).

Reply:
@greatroar We discussed this a bit and would like to remove the entire Versions heading and its paragraphs, except the "Each repository has its own cache sub-directory, consisting of the repository ID
which is chosen at ``init``. All cache directories for different repos are
independent of each other." one which should be kept (instead of the Versions heading).

Reply:
Ok, done.